src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -123,6 +123,14 @@ export interface GetPrismaClientOptions {
   internalDatasources: InternalDatasource[]
 }
 
+// Re-export sql-template-tag
+const sqlTemplateTagExport = (
+  ...args: Parameters<typeof sqlTemplateTag.sqltag>
+) => sqlTemplateTag.sqltag(...args)
+sqlTemplateTagExport.empty = sqlTemplateTag.empty
+sqlTemplateTagExport.join = sqlTemplateTag.join
+sqlTemplateTagExport.raw = sqlTemplateTag.raw
```

Comment:
I would prefer if we could import directly those tags from prisma instead of using the instance to reach them.